app/services/users/create_mascot_account.rb

```diff
@@ -1,29 +1,33 @@
 module Users
   class CreateMascotAccount
-    MASCOT_PARAMS = {
-      email: "mascot@forem.com",
-      username: "mascot",
-      name: "Mascot",
-      profile_image: Settings::General.mascot_image_url,
-      confirmed_at: Time.current,
-      registered_at: Time.current,
-      password: SecureRandom.hex
-    }.freeze
+    MASCOT_PASSWORD = SecureRandom.hex.freeze
```

Comment:
The only thing that changes is that `name` is now translatable. IMO you can keep the original constant around and change the newly added `mascot` method to this:

```ruby
def self.mascot
  @mascot ||= MASCOT_PARAMS.merge(I18n.t("create_mascot.name"))
end
```

Or if we want to keep the changes as minimal as possible, you reuse the existing `merge` call in `mascot_params`:

```ruby
MASCOT_PARAMS.merge(
  name: I18n.t("create_mascot.name")
  password_confirmation: MASCOT_PARAMS[:password])
)
```

That may actually be my preferred solution.